docs/filesystems-file-transfer/filesystems.md

```diff
@@ -43,4 +47,4 @@ OpenMind provides a number of different storage options. See the [OpenMind Docum
 | Vast Scratch | `/om2/scratch/<week-day>` | N/A | Not backed up <br> purged 2 weeks after creation | Scratch space |
 | Lustre Scratch | `/nobackup/scratch/<week-day>` | N/A | Not backed up <br> purged 3 weeks after creation | Scratch space <br>  See [Lustre Best Practices](https://www.nas.nasa.gov/hecc/support/kb/lustre-best-practices_226.html) page |
```

Comment:
Remove the "Lustre Scratch" row in the table of the file system page. It is completely retired now.